docs/zh/quickstart/cli.md

```diff
@@ -0,0 +1,59 @@
+# CLI
+
+CLI，指OpenMLDB的shell客户端，使用发布包`bin`中的binary程序`openmldb`启动，启动命令通常为`bin/openmldb --zk_cluster=127.0.0.1:2181 --zk_root_path=/openmldb --role=sql_client`。本文将介绍常用的一些CLI可配置项与便利的非交互式使用方法，完整的可配置项见`bin/openmldb --help`。
+
```

Comment:
does this script `sbin/openmldb-cli.sh` work?
i saw this in the doc https://openmldb.ai/docs/zh/main/deploy/install_deploy.html#id18

Reply:
if it works, also put this as a shortcut to start CLI